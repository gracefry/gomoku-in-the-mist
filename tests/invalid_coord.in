who
place Z1
who
place A1
who
term
