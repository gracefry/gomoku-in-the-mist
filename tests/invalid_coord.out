B
Invalid coordinate
B
W